buku

```diff
@@ -618,11 +619,15 @@ class BukuDb:
             return
 
         if not self.json:
-            prompt(results, self.noninteractive)
+            if delete is False:
+                prompt_open(results, self.noninteractive)
```

Comment:
If this is the case, I think it's confusing. I agree from the implementation pov it's cleaner, but it's not intuitive.